### PR TITLE
jsonwebtoken - Typings for PR413

### PR DESCRIPTION
**Pull request by Richie765**
See PR https://github.com/auth0/node-jsonwebtoken/pull/413
Please wait until that PR is merged.

### CODE REVIEW

File: types/jsonwebtoken/index.d.ts [hunk 1]
```diff
@@ -83,6 +83,14 @@ export interface SignCallback {
     (err: Error, encoded: string): void;
 }
 
+export interface SecretCallback {
```
Comment (andy-ms) [inline types/jsonwebtoken/index.d.ts]: Don't think this deserves its own type declaration. Can just do:
```ts
export type SecretFunc = (header: object, callback: (err: Error, secretOrPrivateKey: string | buffer) => void) => void;
```

Comment (Richie765) [reply types/jsonwebtoken/index.d.ts]: That's a good one 👍. Will update it soon!

Comment (andy-ms) [reply types/jsonwebtoken/index.d.ts]: Forgot to remove this? It's already been inlined into `SecretFunc` so should be OK to delete.

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

**Comment by minestarks** [comment]
@Richie765 I'm not sure if you're still waiting on something. Do you still need this PR open?

**Comment by minestarks** [comment]
Closing stale PR for housekeeping reasons. If you would still like these changes to get merged, please open a new PR. Thank you!

**Pull request closed without merging**